api-common/src/main/java/io/enmasse/api/auth/KubeAuthApi.java

```diff
@@ -76,7 +76,13 @@ public TokenReview performTokenReview(String token) {
             spec.put("token", token);
             body.put("spec", spec);
 
+            if (log.isDebugEnabled()) {
```

Comment:
is it really worth doing the if(log.isDebugEnabled()) here (and the other places you've used this idiom)?

Reply:
Its done for every api interaction and the responses can be a medium json object, so I thought it might be good to guard it.

Reply:
sure but the JSON object won't get serialized unless isDebugEnabled is true anyway (since that'll be the first check in log.debug), the performance difference is just that between making a method call with no arguments and making a method call with two arguments which I would hope would be negligible

Reply:
Ah you’re right, this is java! :)